intermol/desmond/desmond_parser.py

```diff
@@ -772,28 +793,28 @@ def parse_constraints(self, type, current_molecule_type):
         for j in range(ff_number):
             # water constraints actually get written to rigidwater (i.e. settles) constraints.
 
-            if 'HOH' in entry_values[j] or 'AH' in entry_values[j]:
-                split = entry_values[j].split()
+            if 'HOH' in ev[j] or 'AH' in ev[j]:
```

Comment:
nit: I would keep the slightly more descriptive `entry_values` - already fairly tricky to deduce what these variables represent